### DISCUSSION TITLE
Stack or blend tuned models

### DISCUSSION THREAD

**Discussion by SSMK-wq** [pycaret/pycaret#3133] [Q&A]
I am working on pycaret tool for automated regression.

So, I get the top3 models using `compare_models` and blend them as shown below

    top3 = compare_models(n_select = 3)
    blender = blend_models(top3)

But I would like to blend or stack the tuned models (with best hyparparameters)

So, I tried the below

```
    tuned_knn2 = tune_model(knear,optimize='mae',n_iter = 100)
    tuned_rf2 = tune_model(rf2,optimize='mae',n_iter = 100)
    tuned_ex2 = tune_model(ex2,optimize='mae',n_iter = 100)
```

How can I stack/blend the tuned above models using `blend_models` or `stack_models`?

**Comment by turkalpmd** [comment]
Here is the possible solution about you answer;

```
possible_model_list = [tuned_knn2, tuned_rf2, tuned_ex2]
blender = blend_models(possible_model_list) # this code make blender model
stacker = stack_models(possible_model_list) # this code make stacker model
```
